src/components/nav/nav-controller.ts

```diff
@@ -1442,54 +1442,69 @@ export class NavController extends Ion {
       // the ElementRef of the actual ion-page created
       let pageElementRef = component.location;
 
-      // a new ComponentRef has been created
-      // set the ComponentRef's instance to its ViewController
-      view.setInstance(component.instance);
+      let promise = null;
+      if ( component.instance['_ionicProjectContent'] ) {
```

Comment:
What is _ionicProjectContent for? Can we load the content the same way a tab does?